aptos-move/framework/aptos-framework/sources/voting.move

```diff
@@ -487,6 +487,13 @@ module aptos_framework::voting {
         );
     }
 
+    #[view]
+    /// Return the next unassigned proposal id
+    public fun next_proposal_id<ProposalType: store>(voting_forum_address: address,): u64 acquires VotingForum {
+        let voting_forum = borrow_global_mut<VotingForum<ProposalType>>(voting_forum_address);
```

Comment:
I think we should use `borrow_global` instead of `borrow_global_mut` since this is a view function.

Reply:
at yea good point! I updated the rest of the view functions that also borrowed a mutable reference